CLAUDE.md

```
@@ -0,0 +1,94 @@
+```ts
+type ApiResponse<T> = { success: boolean; data: T | null; error: ApiError | null };
+```
+
+`response.ok` 와 `result.success` 를 함께 확인한다. HTTP 200 이어도 `success: false` 일 수 있다.
+
+## 구조
+
+기능 단위 폴더(`src/features/<기능>/`)로 나누고, 폴더 안을 **타입 / API / 훅 / 화면**
+네 갈래로 쪼갠다. 새 기능도 이 배치를 따른다.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)